src/taipy/gui_core/viselements.json

```diff
@@ -85,10 +85,10 @@
             "doc": "If True, the scenario visualizer can be expanded.<br/>If False, the scenario visualizer is always collapsed and only its name and the <i>submit</i> button are visible."
           },
           {
-            "name": "expanded",
-            "type": "dynamic(bool)",
+            "name": "default_expanded",
```

Comment:
Why the property renaming? That creates what I think is a useless incompatibility...

Reply:
back to expanded